crates/sui-indexer-alt-jsonrpc/src/lib.rs

```diff
@@ -107,13 +116,18 @@ impl RpcService {
             "https://raw.githubusercontent.com/MystenLabs/sui/main/LICENSE",
         );
 
+        let config = RpcConfig {
+            slow_request_threshold: Duration::from_millis(slow_request_threshold_ms),
+            ..Default::default()
+        };
```

Comment:
This seems liable to create a huge confusion -- now there's two configs around, one where we only care about one field, and another where we care about everything else...